dpppt-backend-sdk/dpppt-backend-sdk-ws/src/test/java/org/dpppt/backend/sdk/ws/controller/GaenControllerTest.java

```diff
@@ -846,11 +843,7 @@ public void delayedKeyDateBoundaryCheck() throws Exception {
 			tmpKey.setTransmissionRiskLevel(0);
 			keys.add(tmpKey);
 		}
-		Map<Integer, Boolean> tests = Map.of(-2, false,
-				-1, true,
-				0, true,
-				1, true,
-				2, false);
+		Map<Integer, Boolean> tests = Map.of(-2, false, -1, true, 0, true, 1, true, 2, false);
```

Comment:
In fact I prefer the previous formatting: shows the correspondence of integer -> boolean better.